src/views/Friends.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { useEffect, useState } from 'react';
+import { fetchFriends } from '../services/friends';
+import { useParams } from 'react-router-dom';
+export default function Friends() {
+  const params = useParams();
```

Comment:
We won't need to utilize params in this case, only when we are trying to pull something in from the url, such as an id.